pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -250,6 +250,35 @@ let
       };
     };
 
+  mkDictFromLibreOffice =
+    { shortName
+    , shortDescription
+    , dictFileName
+    , license
+    , readmeFile ? "README_${dictFileName}.txt"
+    , sourceRoot ? dictFileName }:
+    mkDict rec {
+      name = "hunspell-dict-${shortName}-libreoffice-${version}";
+      version = "6.2.0.3";
```

Comment:
I am a bit tentative on using LibreOffice's version here. It is certainly simpler to keep track source updates this way but it doesn't communicate the upstream version number of the dictionary (1.7 for Hungarian in this case)